src/base/bittorrent/sessionimpl.cpp

```diff
@@ -4105,6 +4105,11 @@ void SessionImpl::updateTrackersFromURL()
         {
             if (result.status == Net::DownloadStatus::Success)
             {
+                if (!result.contentType.contains(u"text/plain"_s, Qt::CaseInsensitive))
+                {
+                    LogMsg(tr("Cannot add trackers from URL, expected Content-Type is \'text/plain\' received \"%1\"").arg(result.contentType), Log::WARNING);
```

Comment:
1. Single and double quotes are used inconsistently in this message.
2. I doubt that single quotes need to be escaped.
3. The message format seems inappropriate (and even grammatically invalid).

@Chocobo1?